src/blocks/content-helper/related-top-posts/component-list-item.tsx

```diff
@@ -21,9 +23,17 @@ function RelatedTopPostListItem( { post }: RelatedTopPostListItemProps ): JSX.El
 				<a className="parsely-top-post-stats-link" href={ post.dashUrl } target="_blank" rel="noreferrer" title={ __( 'View in Parse.ly (opens new tab)', 'wp-parsely' ) }>
 					{ post.title }
 				</a>
-				<a className="parsely-top-post-link" href={ post.url } target="_blank" rel="noreferrer" title={ __( 'View Published Post (opens new tab)', 'wp-parsely' ) }>
```

Comment:
Changes `View Published Post` to `View Post` which is more generic.

Just thinking that if users changes the Post from Published to draft then this text should be right.